docs/install/knative-with-operators.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "Installing Knative with Knative Operators"
+weight: 10
+type: "docs"
+---
+
+# Knative installations with Knative Operators
+
+Knative provides operators as tools to install, configure and manage Knative. This guide explains how to install and
+uninstall Knative using Knative operators.
+
+Each component in Knative has a separate operator for installation and configuration. This means that there is a [Serving operator](https://github.com/knative/serving-operator)
+and an [Eventing operator](https://github.com/knative/eventing-operator), and you can choose to install one or both independently.
+
+## Prerequisites:
```

Comment:
Instead of having lots of sub headings and long paragraphs as part of this, I would make the prerequisites a simple bullet list of requirements with links our to other docs as required.

EXAMPLE:

## Prerequisites (or Before you begin)
Knative installation using Operators requires the following:

- Knative v0.13.0 requires a Kubernetes cluster v1.15 or newer, as well as a compatible kubectl. This guide assumes that you've already created a Kubernetes cluster.
- Istio (add link to install docs)